tests/data-files/catalogs/external-child/catalog.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "Catalog",
+  "id": "test",
+  "stac_version": "1.0.0",
+  "description": "test",
+  "links": [
+    {
+      "rel": "self",
+      "href": "/home/jsignell/stactools/catalog.json",
```

Comment:
Shouldn't have user-specific paths in the fixtures.

Reply:
oh gross yeah. Not sure how those snuck in